docs/DeveloperGuide.md

```diff
@@ -258,12 +258,13 @@ _{Explain here how the data archiving feature will be implemented}_
 **Target user profile**:
 
 * has a need to manage a significant number of contacts
+* has a need to aggregate links and social media handles of contacts
 * prefer desktop apps over other types
 * can type fast
 * prefers typing to mouse interactions
 * is reasonably comfortable using CLI apps
```

Comment:
My suggestion:
**Target user profile**:
A university student who
* has a need to organise a significant number of contacts and meetings across various social media platforms
* prefers desktop apps over other types
* can type fast
* prefers typing over mouse
* is reasonably comfortable using CLI apps